pkg/cvo/sync_worker.go

```diff
@@ -648,6 +646,8 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, maxWorkers in
 						Verified:    info.Verified,
 					})
 					return err
+				} else {
+					w.eventRecorder.Eventf(cvoObjectRef, corev1.EventTypeWarning, "PreconditionWarn", "precondition warning for payload loaded version=%q image=%q: %v", desired.Version, desired.Image, err)
```

Comment:
Not sure if/how the semantics of `force` changes once we have warnings but I think this somewhat obfuscates the use of `force`. I'm thinking we should still say that we're "forcing past" everything.

Reply:
I've pushed ff739b2e -> 077d0794 to add a `Forced through blocking failures: ` prefix to the error message when we force through a blocking error.  Does that seem like it would be a clear enough signal?